src/main/java/com/insightfullogic/honest_profiler/ports/sources/FileLogSource.java

```diff
@@ -44,15 +44,25 @@
     public FileLogSource(final File file)
     {
         this.file = file;
+        RandomAccessFile randomAccessFile= null;
         try
         {
-            channel = new RandomAccessFile(file, "r").getChannel();
+            randomAccessFile = new RandomAccessFile(file, "r");
+            channel = randomAccessFile.getChannel();
             remapFile(channel.size());
         }
         catch (IOException e)
         {
             throw new CantReadFromSourceException(e);
         }
+        finally {
+            if(randomAccessFile != null){
+                try {
+                    randomAccessFile.close();
+                }
+                catch(IOException ex){}
+            }
+        }
```

Comment:
Hi, closing the random access file will also close its channel, that we continue to use. Did you run honest profiler after applying this patch?